metricbeat/mb/testing/testdata.go

```diff
@@ -150,14 +151,29 @@ func TestDataFiles(t *testing.T, module, metricSet string) {
 }
 
 // TestDataFilesWithConfig run tests for a testdata config
-func TestDataFilesWithConfig(t *testing.T, module, metricSet string, config DataConfig) {
+func TestDataFilesWithConfig(t *testing.T, module, metricSet string, config DataConfig, testPrefix ...string) {
 	t.Helper()
-	ff, err := filepath.Glob(filepath.Join(config.Path, "*."+config.Suffix))
+
+	// the location of the read files
+	location := filepath.Join(config.Path, "*."+config.Suffix)
+
+	// if this function was called from data_test.go then the testPrefix should be 1
```

Comment:
`testPrefix` is a string that holds the directory prefix, right? How it can 1?

Reply:
Not sure I understand. If this function was called from the test to check that testing is ok (outside the modules), then there is a prefix (see [this](https://github.com/elastic/beats/blob/9d7e4c9ac133e9b578a58848d4f8177b0676f0d4/metricbeat/mb/testing/data/data_test.go#L52)).

Reply:
In the comments it says `if this function was called from data_test.go then the testPrefix should be 1` . How can it be 1?

Reply:
I am still confused, sorry. The line that I linked is this:
`mbtest.TestDataFilesWithConfig(t, moduleName, metricSetName, config, getModulesPath())`
So you can see that there is an extra parameter being passed. That is why the []string has length 1.